### ISSUE TITLE
Contact social media field icon

### ISSUE THREAD

**Issue by fly85** [sulu/sulu#3533]
| Q | A
| --- | ---
| Bug? | no
| New Feature? | yes
| Sulu Version | 1.6.4
| Browser Version | -

#### Actual Behavior

In administration adding a social media field show the fa-user icon in the label

#### Expected Behavior

Show the social media icon (eg. facebook with fa-facebook instead of fa-user)

#### Possible Solutions

- Add a new property/getter/setter `icon` in the SocialMediaProfileType entity
- Add the field mapping in the SocialMediaProfileType.orm.xml
- Set the icon name in the default contact fixtures 
- Change the contact:templates:base.form.html.twig `data-aura-front-icon="user"` to `data-aura-front-icon="{{ defaultSocialMediaProfileType.icon }}"`

**Comment by You85** [comment]
+1

**Comment by wachterjohannes** [comment]
@danrot this could be an addition for the next release.

**Comment by danrot** [comment]
Yeah, that shouldn't be too hard. But I don't think that it will be done for the 1.x series.
